blueprints/layered-profile-server.md

```diff
@@ -93,7 +93,7 @@ With Layered Profiles, hundreds of profiles become a few dozen, each representin
 {
   "response": [{
     "id": 5,
-    "profiles": ["MID", "AMIGA_123", "CDN_FOO"]
+    "profileNames": ["MID", "AMIGA_123", "CDN_FOO"]
```

Comment:
To make it more clear.. I felt profiles was generic.

Reply:
Idk, I think the typing makes it clear. IDs are numbers, names are strings. Typically we use `profileID` and `profileName` because objects have _both_, so you need to be more specific.

I, personally, like `profiles` better, but it doesn't really matter. `profileNames` is fine; change it or don't, but it should be consistent everywhere (i.e. in the Go Struct - I think I have a comment about that later).